pennylane/ops/qubit.py

```diff
@@ -2051,6 +2080,107 @@ def __init__(self, control_wires=None, wires=None, control_values=None, do_queue
             do_queue=do_queue,
         )
 
+    # pylint: disable=unused-argument
+    def decomposition(self, *args, **kwargs):
```

Comment:
Note that `decomposition` is not a static method. We have access to `self` and also don't care about `*args, **kwargs` because we read the relevant data from the object's state (e.g., `self.control_wires`).

This is going against the standard, but it seems to work.